main.go

```diff
@@ -301,7 +304,6 @@ func (cmd *mainCmd) Run(args []string) error {
 			errors = append(errors, fmt.Errorf("failed to rewrite %q: %v", filename, err))
 			continue
 		}
-
 		bs, err := imports.Process(filename, out.Bytes(), &imports.Options{
```

Comment:
If we're not going to use the results for imports.Process, could we skip the call to imports.Process altogether?

Reply:
We would be using the result of imports.Process for cases when the flag isn't passed, 
which maintains gopatch's earlier formatting rule that follows go's formatting. I


For a cleaner look I'll switch the if clause, to have the imports.Process only be called if DisableGroupImports flag is not passed 
```
bs := out.Bytes()
if !opts.DisableGroupImports {
                bs, err = imports.Process(filename, out.Bytes(), &imports.Options{
                         Comments:   true,
			TabIndent:  true,
                         TabWidth:   8,
			FormatOnly: true,
	        })
	        if err != nil {
			log.Printf("%s: failed: %v", filename, err)
			errors = append(errors, fmt.Errorf("reformat %q: %v", filename, err))
			continue
	        }
}
```